### DISCUSSION TITLE
GitHub pages site not appearing in any search engine

### DISCUSSION THREAD

**Discussion by kosalnith** [community/community#30723] [Other Feature Feedback, Questions, & Ideas]
Hi GitHub Community,

I changed my user name in GitHub, so my GitHub page also changed the name too. But the problem is that my new site which is hosted on GitHub page which isn't appearing on any search engine until today since I changed the name among 3 months. At the same time, I also verified this site in Google Console through the HTML tag. However, it works just by going to the URL but it does not show in Google search.

The site is http://kosalnith.github.io

Can anyone help?

Best,
Kosal

**Comment by radar-blip** [comment]
You might want to do some search engine optimization. Your site may not be indexed by google yet, 3 months isnt that long considering all the websites out there.
Here is a good reference for SEO: https://developers.google.com/search/docs/beginner/seo-starter-guide?hl=en%2F&visit_id=637969894813320757-1850286758&rd=1

**Reply by kosalnith** [reply]
Thanks for your help! I will read it to understand. Did you have any brief instructions to do that? Thanks!

**Reply by radar-blip** [reply]
It would help you more just to read it yourself. You will understand it better that way. 
And, it will be worth it.

**Reply by kosalnith** [reply]
Sounds good!

**Comment by jfullstackdev** [comment]
in the past, out of curiosity, I did all of this to actually be indexed by Google, even Google Search Console and Analytics

one tip is that, be active in social medias, the more presence, the more it will be recognized by web crawlers 

and take note, your username must be consistent across sites, the more you share links and others are clicking them, the more chances you will be indexed by Google the shortest time possible, follow as much users as possible, it's just like you advertising your own decentralized links

**Reply by kosalnith** [reply]
Thank you! It would a good strategy.
